### DISCUSSION TITLE
[Question] Notes tree / sidebar on the right side?

### DISCUSSION THREAD

**Discussion by rdtm443** [TriliumNext/Trilium#6325] [Q&A] [answered]
Hello folks

Is it possible to have the side panel on the right side? 

As I am left handed, it is very difficult to take canvas notes (i.e. draw), as I am constantly hitting some buttons on the sidepanel with my palm.

Thanks!

Best,

rdtm443

**Comment by zadam** [comment]
Hello, no, this is not possible.

**Comment by rdtm443** [comment]
> Hello, no, this is not possible.

Hey there

..that's a pity. Because, just to say, in obsidian there is an excalidraw plugin that lets you select left hand mode. I thought maybe it is possible as well with trilium (as it also is partly based on excalidraw). 

Or, maybe it is possible to move the settings panel within the canvas to the right?

Otherwise it is not possible to use canvas as left handed person, unfortunately :-/

Best,

rdtm443

**Comment by meichthys** [comment]
@rdtm443 Your best bet may be to keep the sidebar collapsed:
![image](https://user-images.githubusercontent.com/10717998/228060910-0d0c9100-dd20-4c15-a9b0-6a6ff01463ec.png)


Alternatively, you could rotate your tablet/device make your drawing, then when youre finished, just rotate your entire drawing with a `ctrl-a` and then grab the dot at the top of the selection and rotate (if you hold shift while rotating the selection, it will be easier to snap the rotation to 90 or 180 degrees):
![image](https://user-images.githubusercontent.com/10717998/228061151-c35bd21f-87d6-492c-8ccc-e91b7b770d6e.png)

**Comment by rdtm443** [comment]
@meichthys Hey there, thanks for the reply. Collapsing the sidebar is helping a little bit at least, but then still the toolbar is in the way. In excalidraw it is movable i think, but not so in trilium?

![image](https://user-images.githubusercontent.com/25002690/228145012-37b6f59c-15f5-426c-829e-dbd2c1e17897.png)

**Comment by zerebos** [comment, marked as answer]
Can't this be done with css?
```css
#left-pane {
    order: 3;
}
```
Seems to work for me. Obviously it's not ideal with the titlebar and everything, but as a temporary measure it seems fine.

![image](https://github.com/zadam/trilium/assets/6865942/a6e06e95-6bac-4fb6-bcf4-d4612e2164b6)

**Comment by CobriMediaJulien** [comment]
You can change the appearance of the sidebar now:

<img width="1653" height="254" alt="image" src="https://github.com/user-attachments/assets/4680e780-be2a-4133-95af-9d74bbbc631f" />
